tools/shoestring/shoestring/internal/Preparer.py

```diff
@@ -163,12 +163,15 @@ def create_subdirectories(self):
 			self.directories.userconfig,
 			self.directories.resources
 		]
+
 		if NodeFeatures.API in self.config.node.features:
-			directories.append(self.directories.dbdata)
-			directories.append(self.directories.rest_cache)
 			if self.config.node.api_https:
 				directories.append(self.directories.https_proxy)
 
+		if self.config.node.full_api:
+			directories.append(self.directories.dbdata)
+			directories.append(self.directories.rest_cache)
```

Comment:
I think ``rest_cache`` is used in the light node.  🤔

Reply:
actually, what is the `rest_cache` folder doing?

Reply:
This should be the REST working folder.

```

  light-rest-api:
    image: '{{ catapult_rest_image }}'
    user: '{{ user }}'
    command: npm run start-light --prefix /app /userconfig/rest.json
    environment:
      - HOME=/symbol-workdir
      - NODE_ENV=production
    working_dir: /symbol-workdir
{%- if not api_https %}
    ports:
      - 3000:3000
{%- endif %}
    volumes:
      - ./startup:/startup:ro
      - ./userconfig:/userconfig:ro
      - ./keys/cert:/certificates:ro
      - ./data:/data
      - ./logs:/logs
      - ./rest-cache:/symbol-workdir
```